code.gs

```diff
@@ -1513,10 +1516,19 @@ function beautifyLabel (label) {
     case 'WORK_FAX':
     case 'GOOGLE_VOICE':
     case 'PAGER':
+    case 'OTHER_PHONE': // Fake label for output.
+    /*
+     * (falls through)
```

Comment:
I would suggest leaving 'falls through' in brackets as the main reason for these comments is to group labels, not about 'falls through'.